sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -262,14 +262,17 @@ scatter(Tx *p, Toffset offset, simd<Tx, N> vals, simd_mask<N> mask = 1) {
   scatter<Tx, N>(p, simd<Toffset, N>(offset), vals, mask);
 }
 
-/// Loads a contiguous block of memory from given memory address and returns
-/// the loaded data as a vector. Actual code generated depends on the
-/// alignment parameter.
+/// Loads a contiguous block of memory from the given memory address \p addr
+/// and returns the loaded data as a vector.
+/// The generated code depends on the combination {T, N, Flags}.
+/// Providing flags specifying the alignment of 16-bites or more produces more
```

Comment:
bites->bytes, same typo in a few spots

Reply:
Fixed in 2241ceea91cbed7c4dbbcafa0ef201fe706a8097. Thank you